README.md

```diff
@@ -174,9 +174,93 @@ only works using a custom runner hosted on GCP.**
 The action will automatically detect and use the Application Default
 Credentials.
 
+## Example Workflows
+
+* [Deploy a prebuilt container](#deploy-a-prebuilt-container)
+
+* [Build and deploy a container](#build-and-deploy-a-container)
+
+### Setup
+
+1.  Create a new Google Cloud Project (or select an existing project).
+
+1. [Enable the Cloud Run API](https://console.cloud.google.com/flows/enableapi?apiid=run.googleapis.com).
+
+1.  [Create a Google Cloud service account][sa] or select an existing one.
+
+1.  Add the the following [Cloud IAM roles][roles] to your service account:
+
+    - `Cloud Run Admin` - allows for the creation of new Cloud Run services
+
+    - `Service Account User` -  required to deploy to Cloud Run as service account
+
+    - `Storage Admin` - allow push to Google Container Registry
```

Comment:
This is broad at the project level, perhaps we can link to further reading [here](https://cloud.google.com/container-registry/docs/access-control#grant) for best practices of granting at the bucket level.

Reply:
sgtm